src/parser.js

```diff
@@ -70,13 +70,8 @@ define([
             title = head.children('h\\:title, title'),
             binds = head.find('bind'),
             instances = _getInstances(xml),
-            data = $(instances[0]).children();
-
-        xml.find('setvalue').each(function () {
-            var $this = $(this);
-            form.addSetValue(
-                $this.attr('event'), $this.attr('ref'), $this.attr('value'));
-        });
+            data = $(instances[0]).children(),
+            setValues = xml.find('setvalue');
```

Comment:
Hmm. I think it would be a good idea to change it to find immediate children, but can wait until we hear from @ctsims 

Maybe we could warn on certain node ids like bind or setvalue (I think you had a recent PR that did that)?